### DISCUSSION TITLE
Firebase adapter Module not found: Can't resolve 'fs' ?

### DISCUSSION THREAD

**Discussion by ImamJanjua** [nextauthjs/next-auth#11023] [Help]
Hello, I have set up all as it been described in the docs but getting this weird error for node modules

<img width="1440" alt="Bildschirmfoto 2024-05-27 um 16 26 18" src="https://github.com/nextauthjs/next-auth/assets/137432044/be53b4dd-47cc-49da-afb8-565b893d3165">

      "@auth/firebase-adapter": "^2.1.0",
  "firebase-admin": "^11.11.1",
        "next-auth": "^5.0.0-beta.18",
macOs
node 20
chrome

**Comment by k-taro56** [comment]
Are you using `export const runtime = "edge"`?

**Comment by ImamJanjua** [comment]
@k-taro56  no I am not using edge
